src/chains/ethereum/ethereum/src/provider.ts

```diff
@@ -164,6 +164,7 @@ export class EthereumProvider
       providerOptions.fork.url ||
       providerOptions.fork.provider ||
       providerOptions.fork.network;
+
```

Comment:
I'll remove this but waiting for feedback so that I can couple it with any additional changes.